.github/workflows/lightouse-ci.yaml

```
@@ -0,0 +1,112 @@
+name: Run lighthouse CI When Push
+on:
+  pull_request:
+    branches: [dev]
+    paths: ["frontend/lighthouserc.js"]
```

```suggestion
  pull_request:
    branches: [dev]
    paths: ["frontend/**"]
    types: [opened, synchronize, reopened]
```
dev branch로부터의 pr에서만 동작, "frontend/**" 하위의 모든 변경점에 대한 동작, pr 오픈/pr 재요청/pr 리오픈에 대해서 동작
으로 변경하면 조금더 상황에 맞는 ci action이 돌아갈 것 같네요 :)

---

네 수정했습니다.